### PR TITLE
feat: discord integration

### PR DESCRIPTION
**Pull request by bmassemin**
This PR provides configuration for Discord integration.
Today it is possible to enable Discord using the `config.configYml`, but there's no option for adding configuration from a secret.

### CODE REVIEW

File: sentry/values.yaml
Comment (Mokto) [inline sentry/values.yaml:1]: Thanks. Can you please remove the auto formatting in this file ?

Comment (bmassemin) [reply sentry/values.yaml:1]: oops my bad, vscode always tricks me

Comment (bmassemin) [reply sentry/values.yaml:1]: done!

**Review by Mokto — APPROVED** [review]

File: sentry/templates/_helper.tpl [hunk 1]
```diff
@@ -560,6 +560,28 @@ Common Sentry environment variables
...
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.discord.existingSecret }}
+      key: {{ default "client-secret" .Values.slack.existingSecretClientSecret }}
+- name: DISCORD_BOT_TOKEN
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.discord.existingSecret }}
+      key: {{ default "bot-token" .Values.slack.existingSecretBotToken }}      
+{{- end }}
```
Comment (ollewobbla) [inline sentry/templates/_helper.tpl:584]: Shouldn't these keys point to .Values.discord instead of Values.slack?

Comment (bmassemin) [reply sentry/templates/_helper.tpl:584]: oof sorry, bad copy paste

Comment (bmassemin) [reply sentry/templates/_helper.tpl:584]: https://github.com/sentry-kubernetes/charts/pull/1408

**Pull request merged**